nixos/doc/manual/installation/changing-config.chapter.md

```diff
@@ -55,6 +55,14 @@ which causes the new configuration (and previous ones created using
 This can be useful to separate test configurations from "stable"
 configurations.
 
+A repl, or read-eval-print loop, is also available. You can inspect your configuration and use the Nix language with
+
+```ShellSession
+# nixos-rebuild repl
+```
+
+Your configuration is loaded into the `config` variable. Use tab for autocompletion, use the `:r` command to reload the configuration files. See `:?` or [`nix repl` in the Nix manual](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-repl.html) to learn more.
```

Comment:
Either I do not understand how ``:r`` is supposed to work or it is broken. For me it just clears the loaded variables and loads --file again and totally ignored ``:lf``.

Reply:
`:lf` behaves different from `:l`, and taken together I consider it to be quite broken, yes.
- https://github.com/NixOS/nix/issues/8753